samples/general/mobile_nerf/README.adoc

```diff
@@ -17,15 +17,16 @@
 -
 ////
 
-= NeRF
+= Mobile NeRF
 
 ifdef::site-gen-antora[]
-TIP: The source for this sample can be found in the https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/general/nerf[Khronos Vulkan samples github repository].
+TIP: The source for this sample can be found in the https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/general/mobile_nerf[Khronos Vulkan samples github repository].
 endif::[]
 
 This sample is a modified version of the https://mobile-nerf.github.io/[Mobile Nerf] developed by Google. 
 It's based on its original https://github.com/google-research/jax3d/tree/main/jax3d/projects/mobilenerf[source code] but optimized for Vulkan.
-
+This is a different version from the traditional NeRF as it uses the rasterization pipeline to render the final image.
```

Comment:
Can you elaborate on this? At least for me personally, it's not clear what a "traditional" NeRF is and what "mobile" means in that context.

Reply:
Just updated the readme with a better description. 

The idea is that a traditional NeRF would require tracing multiple rays and querying a MLP hundreds of times for each ray, which can barely provide interactive FPS even on recent desktop GPUs.

The mobile version uses a mesh + texture combo, which is normally rasterized, and run a single time through a small MLP for each of its visible pixels.

Another reason to change back to mobile nerf is that this implementation is based on an existing paper which names this alternative technique (of rendering a NeRF) as Mobile NeRF.